lib/reporters/html/partials/cov.html

```diff
@@ -6,6 +6,10 @@
         <div class="hits">{{coverage.cov.hits}}</div>
         <div class="misses">{{coverage.cov.misses}}</div>
     </div>
+    <div id="filters">
+      <input type="checkbox" checked="" onchange="filter(this)" value="generated" id="show-generated">
+      <label for="show-generated">Show generated files</label>
+    </div>
```

Comment:
From now, reporter displays both original and generated sources.

This toggle allows to show/hide generated files.